tree/treeplayer/src/TRootDS.cxx

```diff
@@ -139,7 +139,11 @@ void TRootDS::SetNSlots(unsigned int nSlots)
 
 TDataFrame MakeRootDataFrame(std::string_view treeName, std::string_view fileNameGlob)
 {
+#ifdef R__WIN32
+   ROOT::Experimental::TDataFrame tdf(std::unique_ptr<TRootDS>(new TRootDS(treeName, fileNameGlob)));
+#else
    ROOT::Experimental::TDataFrame tdf(std::make_unique<TRootDS>(treeName, fileNameGlob));
+#endif
```

Comment:
I have the feeling this ifdef clutters the code a bit. It's a compiler bug workaround.
If it's not possible to fix the make_unique template, I would propose to remove the ifdef, move to the unique_ptr ctor and add a clear comment stating that make_unique does not work yet on windows.

Reply:
Wait, I think I found the origin of the problem...